app/app/static/custom/ourscript.js

```diff
@@ -37,6 +37,7 @@ const addToQueueList = async (data) => {
 		// There's an error
 		console.log(error)
```

Comment:
Not sure if within the scope of this pull request, but just pointing out. If the fetch request failed because lost internet connection, there will be no feedback aside from the console. maybe in the future we should add a feedback about it

Reply:
Yeah not something that I added in this pull request but something to look into